style.css

```diff
@@ -231,6 +223,39 @@ footer {
   justify-content: space-between;
 }
 
+/* a mini theme for blockly as it bugged me */
+.blocklyTreeLabel {
+  font-family: var(--theme-font--display);
+  color: var(--theme-color--ink);
+}
+.blocklyToolboxDiv,
+.blocklyFlyoutBackground {
+  background: var(--theme-color--paper);
+  color: var(--theme-color--ink);
+  fill: var(--theme-color--paper);
+}
+
+.blocklyToolboxDiv {
+  border: var(--theme-border);
+  padding: 20px 10px;
+  box-shadow: 2px 6px 0 var(--theme-color--pop);
+}
+.blocklyMainBackground {
+  fill: transparent !important;
+  border: var(--theme-border);
+  stroke: var(--theme-color--accent);
+  stroke-width: 0;
```

Comment:
```suggestion
```
I think showing where the boundary of the box is makes it a bit more crowded, but more obvious that there is a box where things can be placed? Alternatively, play with the transparency>